mitodl/bootcamp-ecommerce
# defer enrollment

#### What are the relevant tickets?
fixes #1208 

#### What's this PR do?
defer enrollment

#### How should this be manually tested?
just run the management command and check for the defer enrollment

`docker-compose exec web python manage.py defer_enrollment --user=admin --from-run=bootcamp-v1:private+MCR-f2f+R2 --to-run=bootcamp-v1:public+DER-f2f+R2 --order=2 -f`

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
<img width="845" alt="Screenshot 2021-04-29 at 15 37 32" src="https://user-images.githubusercontent.com/4043989/116550605-6b516780-a910-11eb-893c-10e78522b79f.png">
<img width="843" alt="Screenshot 2021-04-29 at 15 37 41" src="https://user-images.githubusercontent.com/4043989/116550630-70161b80-a910-11eb-8dd8-348c386b1d20.png">
<img width="845" alt="Screenshot 2021-04-29 at 15 44 23" src="https://user-images.githubusercontent.com/4043989/116550635-71474880-a910-11eb-8007-a7ac321da9b1.png">
<img width="847" alt="Screenshot 2021-04-29 at 15 49 59" src="https://user-images.githubusercontent.com/4043989/116550641-71dfdf00-a910-11eb-8baa-8440eb373b6d.png">
<img width="845" alt="Screenshot 2021-04-29 at 16 08 44" src="https://user-images.githubusercontent.com/4043989/116550647-73110c00-a910-11eb-9fb3-f8ec31099466.png">
<img width="843" alt="Screenshot 2021-04-29 at 16 15 56" src="https://user-images.githubusercontent.com/4043989/116550651-73a9a280-a910-11eb-91a6-2923d53281cf.png">
<img width="1168" alt="Screenshot 2021-04-29 at 16 16 43" src="https://user-images.githubusercontent.com/4043989/116550657-75736600-a910-11eb-9c3b-d242690a6905.png">
<img width="847" alt="Screenshot 2021-04-29 at 16 33 42" src="https://user-images.githubusercontent.com/4043989/116550665-760bfc80-a910-11eb-98e6-0f842b6f5650.png">
<img width="846" alt="Screenshot 2021-04-29 at 16 34 17" src="https://user-images.githubusercontent.com/4043989/116550667-76a49300-a910-11eb-8fef-9c7962c741ec.png">
<img width="849" alt="Screenshot 2021-04-29 at 16 53 55" src="https://user-images.githubusercontent.com/4043989/116550681-7b694700-a910-11eb-884b-b9cf3d6f51b5.png">
<img width="846" alt="Screenshot 2021-04-29 at 17 10 30" src="https://user-images.githubusercontent.com/4043989/116550686-7efcce00-a910-11eb-8e2a-4ab24dff04d6.png">
<img width="844" alt="Screenshot 2021-04-29 at 17 10 45" src="https://user-images.githubusercontent.com/4043989/116550689-802dfb00-a910-11eb-8cbf-76b5e805ceda.png">
<img width="1158" alt="Screenshot 2021-04-29 at 17 15 51" src="https://user-images.githubusercontent.com/4043989/116550691-80c69180-a910-11eb-8bfa-16fbe8854712.png">
<img width="844" alt="Screenshot 2021-04-29 at 17 21 35" src="https://user-images.githubusercontent.com/4043989/116550694-815f2800-a910-11eb-955d-8044cacc819b.png">
